src/main/java/com/ebivariation/contigalias/controller/AdminController.java

```diff
@@ -42,7 +43,8 @@ public AdminController(AssemblyService service) {
         this.service = service;
     }
 
-    @GetMapping(value = "assemblies/{accession}")
+    @ApiOperation(value = "Get or fetch an assembly using its Genbank or Refseq accession.")
+    @GetMapping(value = "v1/assemblies/{accession}", produces = "application/json")
     public ResponseEntity<Optional<AssemblyEntity>> getAssemblyOrFetchByAccession(
             @PathVariable String accession) throws IOException {
```

Comment:
Please use `@ApiParam` annotation to add description and examples for the fields so its clearer to users. You can see implementations of this [here](https://github.com/EBIvariation/eva-accession/blob/master/eva-accession-ws/src/main/java/uk/ac/ebi/eva/accession/ws/rest/ClusteredVariantsRestController.java#L183)
```suggestion
            @ApiParam("assembly accesion in GCA format, e.g.: GCA_000003055.3") @PathVariable String accession) throws IOException {
```